Makefile

```diff
@@ -61,7 +61,7 @@ COMMON_CFLAGS += -iquote $(srcdir) -iquote $(objdir) -iquote $(srcdir)/arch/$(AR
 COMMON_CFLAGS += -W -Wall -Wno-unused-parameter -Wno-missing-field-initializers
 COMMON_CFLAGS += -Wdeclaration-after-statement -Wstrict-prototypes
 
-COMMON_LDFLAGS := -ldl -pthread -Wl,-z,noexecstack $(LDFLAGS)
+COMMON_LDFLAGS := -ldl -lm -pthread -Wl,-z,noexecstack $(LDFLAGS)
```

Comment:
I don't think we need the math library for libmcount or other places.  Please add it to `UFTRACE_LDFLAGS` instead.

Reply:
Thanks for the suggestions. I'm removing it to `UFTRACE_LDFLAGS`!